networkx/algorithms/community/modularity_max.py

```diff
@@ -40,27 +42,18 @@ def greedy_modularity_communities(G, weight=None, resolution=1, n_communities=1)
         If resolution is less than 1, modularity favors larger communities.
         Greater than 1 favors smaller communities.
 
-    n_communities: int
-        Desired number of communities: the community merging process is
-        terminated once this number of communities is reached, or until
-        modularity can not be further increased. Must be between 1 and the
-        total number of nodes in `G`. Default is ``1``, meaning the community
-        merging process continues until all nodes are in the same community
-        or until the best community structure is found.
+    Yields
+    ------
+    Alternating yield statements produce the following two objects:
 
-    Returns
-    -------
-    partition: list
-        A list of frozensets of nodes, one for each community.
-        Sorted by length with largest communities first.
+    communities: dict_values
+        A dict_values of frozensets of nodes, one for each community.
+        This represents a partition of the nodes of the graph into communities.
+        The first yield is the partition with each node in its own community.
 
-    Examples
-    --------
-    >>> from networkx.algorithms.community import greedy_modularity_communities
-    >>> G = nx.karate_club_graph()
-    >>> c = greedy_modularity_communities(G)
-    >>> sorted(c[0])
-    [8, 14, 15, 18, 20, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33]
+    dq: float
+        The change in modularity when merging the next two communities
+        that lead to the largest modularity.
```

Comment:
```suggestion
        that leads to the largest modularity.
```
I think, or maybe "will lead" is better - the choice is yours!